app/assets/javascripts/hub.coffee

```diff
@@ -17,9 +17,11 @@ $(document).on 'turbolinks:load', ->
   #-----------------------
   if window.location.hash
     hash = window.location.hash.substring(1)
-    $('html, body').animate {
-      scrollTop: ($('a[name="'+hash+'"]').offset().top) + 15
```

Comment:
@deekoder Conditional hash check is added to prevent anchor seeking on non hash URLs. Currently if you go to https://docs.minio.io/hub/index, there is a javascript error which prevents the functionality of search and some other JS functions. Using this fix, all the non hash URLs will be ignored. 

This has been tested locally and working.